hw5/hw5/ViewController.swift

```diff
@@ -0,0 +1,70 @@
+//
+//  ViewController.swift
+//  hw5
+//
+//  Created by Илья Мунгалов on 15.03.2022.
+//
+
+import UIKit
+
+class ViewController: UIViewController {
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        // Создать словарь, где ключ - название города/страны, значение - интересный факт и континент, где распорожен город/страна (подумайте, какой тип данных лучше использовать для значения).
+        var cityDictionary: [String: [String]]
```

Comment:
I believe storing a fact and continent in the `String` array will cause a problem in the third task. Also, this way will be able to forget to put either fact or continent or both. So it's better to think about other type to store both continent and fact